docs/_includes/tk-photoshopcc

```diff
@@ -1 +1 @@
-2019 — 2022
\ No newline at end of file
+2019 — 24.1.1
```

Comment:
For Adobe, we need to decide whether we use year versions of version numbers.
As a user, I think you see the year's version first.
I suggest doing both: `2023 (v24.1.1)`

Could you align the same logic on oldest version / newest version, AE and PS?